server/resolve.js

```diff
@@ -27,18 +27,22 @@ function getPaths (id) {
   const i = sep === '/' ? id : id.replace(/\//g, sep)
 
   if (i.slice(-3) === '.js') return [i]
+  if (i.slice(-4) === '.jsx') return [i]
   if (i.slice(-5) === '.json') return [i]
 
   if (i[i.length - 1] === sep) {
     return [
       i + 'index.js',
+      i + 'index.jsx',
```

Comment:
Do we need this?

Reply:
It's used by the ondemandentries plugin to resolve the file within webpack. That's why it's needed here.

Reply:
next build -> next start etc work fine without this.